### PR TITLE
PCHR-2084: Fix namespace

### PR DESCRIPTION
**Pull request by AkA84**
**This is only a temporary solution that will be scrapped once the Bootstrap consolidation takes place**

# Problem
The T&A extension was relying on the [ui-select](https://github.com/angular-ui/ui-select) style contained in the `org.civicrm.bootstrapcivicrm` theme in order to have the dropdowns styled correctly (although, for some reason, it was also using a local copy of the style, plus some customizations).

The theme though is namespaced, and given that T&A still don't use that theme fully (it has its own copy of bootstrap still) and that it's not namespaced, ad-hoc adjustments to the theme had to be made in order for the ui-select styles to still come through, which lead to the problems described in https://github.com/civicrm/civihr/pull/1641 and related PRs

# Solution
As a *temporary* solution, the T&A theme is now referencing directly the `ui-select` and `civihr-ui-select` (the ui-select custom theme) partials contained in the `org.civicrm.bootstrapcivicrm` extension
```scss
// civitasks.scss

// Applies the #civitasks namespace
##{nth($module-list, 1)} {
  // ...
  @import "civitasks/modules/civihr-ui-select";
}
```
```scss
// civitasks/modules/_civihr-ui-select.scss
@import "../../../../../civihr/org.civicrm.bootstrapcivicrm/scss/bootstrap/vendor/ui-select/ui-select";
@import "../../../../../civihr/org.civicrm.bootstrapcivicrm/scss/bootstrap/components/civihr-ui-select/civihr-ui-select";
```
Additionally, all the T&A ad-hoc customizations made on the common theme, are now placed in the T&A theme instead
```scss
// civitasks/modules/_civihr-ui-select.scss

// Customizations specific for T&A
.civihr-ui-select {
  // ...
}
```
Last problem was that the two partials in the `org.civicrm.bootstrapcivicrm` extension were referencing SASS variables that have a different values between the common theme and the T&A theme, so as a *temporary* solution those variables had been reassigned new values so that they match the values in the common theme

```scss
// civitasks/modules/_civihr-ui-select.scss

// Override variables so they match the values in the shoreditch theme
$dropdown-border:        #727E8A;
$input-border-radius:    0;
$brand-primary:          #42AFCB;
$brand-danger:           #E6807F;
$gray-lighter:           #F3F6F7;
$gray-dark:              #727E8A;
```

Finally, the local copy of the ui-select theme and its customizations had been removed

### CODE REVIEW

File: uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/modules/_civihr-ui-select.scss [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  & a.select2-search-choice-close {
+    top: 0 !important;
+  }
+
+  &.open {
+
+    .select2-chosen {
+      border: 1px solid $dropdown-border !important;
+      padding-left: 12px !important;
+      border-bottom: 0 !important;
```
Comment (deb1990) [inline uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/modules/_civihr-ui-select.scss]: Please arrange the properties alphabatically

File: uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/modules/_civihr-ui-select.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  .select2-drop-active {
+    @extend %ui-select-border;
+    border-top: 0 !important;
+  }
+
+  &.ui-select-multiple {
+    &:not(.open) {
+      .select2-choices::before {
+        content: '\f002' !important;
```
Comment (deb1990) [inline uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/modules/_civihr-ui-select.scss]: Save the icon value in a variable with proper name(eg. `down-arrow`) and use. This is to increase readability.

File: uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/modules/_civihr-ui-select.scss (hunk 1 above)
Comment (AkA84) [inline uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/modules/_civihr-ui-select.scss]: Done

File: uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/modules/_civihr-ui-select.scss (hunk 2 above)
Comment (AkA84) [inline uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/modules/_civihr-ui-select.scss]: I did that, but the output was different
```scss
content: "" !important;
```
i've timeboxed it for 10 mins to figure out why and couldn't, so i just skipped it. Not worth the time considering this is all temporary

**Pull request merged**